arroyo-worker/src/formats/mod.rs

```diff
@@ -180,22 +180,22 @@ impl<T: SchemaData> DataSerializer<T> {
     pub fn to_vec(&self, record: &T) -> Option<Vec<u8>> {
         match &self.format {
             Format::Json(json) => {
-                let v = if json.include_schema {
+                let mut writer: Vec<u8> = vec![];
+                if json.confluent_schema_registry {
+                    writer.push(0);
+                    writer.extend(json.confluent_schema_version.expect("must have computed schema version to write using confluent schema registry").to_be_bytes());
+                }
+                if json.include_schema {
```

Comment:
I don't think confluent_schema_registry and include_schema are really compatible — they're two different ways to communicate the schema to kafka consumers

Reply:
Happy to prevent this. Have an idea of where we should be validating?

Reply:
upfront, in JsonFormat::from_opts